Backend/src/main/java/idealab/api/controller/EmailMessageController.java

```diff
@@ -0,0 +1,30 @@
+package idealab.api.controller;
+
+import idealab.api.dto.request.EmailMessageUpdateRequest;
+import idealab.api.dto.response.EmailMessageResponse;
+import idealab.api.operations.EmailMessageOperations;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+@RestController
+@RequestMapping("/api/message")
+public class EmailMessageController {
+    private final EmailMessageOperations emailMessageOperations;
+
+    public EmailMessageController(EmailMessageOperations emailMessageOperations) {
+        this.emailMessageOperations = emailMessageOperations;
+    }
+
+    @GetMapping()
+    public ResponseEntity<?> getMessageByStatusController(@RequestParam(required = true) String status) {
+        EmailMessageResponse response = emailMessageOperations.getMessageByStatus(status);
+        return new ResponseEntity<>(response, response.getHttpStatus());
+    }
+
+    @PutMapping()
```

Comment:
I am getting this error: 
![image](https://user-images.githubusercontent.com/48397717/70380731-e7eb3c80-18fc-11ea-9ab8-2ad410366d8a.png)
Am I doing anything wrong?

Reply:
@hannahkamundson I did this under a new `/api/message` endpoint, we can change that if you want.

Reply:
Got it to work! Thanks! I must've typed the wrong thing :)